tools/ui-components/package.json

```
@@ -75,7 +75,7 @@
     "build-storybook": "build-storybook",
     "build": "pnpm run build:css && pnpm run build:js && tsc",
     "build:js": "cross-env NODE_ENV=production rollup -c",
-    "build:css": "pnpm dlx tailwindcss -i ./src/base.css -o ./dist/base.css --minify",
+    "build:css": "npx -y tailwindcss -i ./src/base.css -o ./dist/base.css --minify",
```

```suggestion
    "build:css": "pnpm exec -y tailwindcss -i ./src/base.css -o ./dist/base.css --minify",
```

My understanding is that `pnpm exec` will also look for local files - should we use that instead, to maintain parity with the rest of our build environment?

---

`pnpm` is good, we just don't have to offer everything it offers. We can use the best parts.